docs/docsite/rst/dev_guide/style_guide/preferred_terms.rst

```diff
@@ -0,0 +1,58 @@
+.. _preferred_terms:
+
+Preferred terminology
+=====================
+
+.. contents::
+  :local:
+
+The following is the list of preferred terms for Ansible ecosystem projects.
+
+.. glossary::
+
+   Ansible
+   Use on all occurrences or ``ansible`` if referring to the CLI command. Preference is to use this to refer to the Ansible community package.
```

Comment:
The definitions under each term should have extra indentation per https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#glossary
```suggestion
      Use on all occurrences or ``ansible`` if referring to the CLI command. Preference is to use this to refer to the Ansible community package.
```

Reply:
ok thanks. Frankly I was so mortified by all my spelling errors I forgot to take a closer look at how the glossary works. Thanks

Reply:
I tend to use https://github.com/sphinx-contrib/spelling for detecting typos...

Reply:
Personally I like codespell https://github.com/codespell-project/codespell

Reply:
I didn't know it can check RST.

Reply:
Ok now I have a duplicate term definition problem because we already have an actual glossary in the appendix.  I'm inclined to remove the glossary from this section because it's not really defining the terms, just how to type them in documentation.